src/swap/SwapScreen.tsx

```diff
@@ -725,7 +721,10 @@ export function SwapScreen({ route }: Props) {
             <InLineNotification
               severity={Severity.Warning}
               title={t('swapScreen.noUsdPriceWarning.title', { localCurrency })}
-              description={t('swapScreen.noUsdPriceWarning.description', { localCurrency })}
+              description={t('swapScreen.noUsdPriceWarning.description', {
+                localCurrency,
+                tokenSymbol: toToken?.symbol,
+              })}
```

Comment:
super nit: do we need `?`?

Reply:
yes, toToken can be undefined. in reality, it won't be if this warning is displayed. i either need optional chaining here or to use `!` (i think optional chaining is less bad)